ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -20,7 +21,8 @@
 class TtsTask:
     implementation: str = simple_parsing.field(default="azure", alias="-i")
     # Column name containing the text to convert to audio. It can be a Jinja variable expression.
-    column_name: str = simple_parsing.field(default="question", alias="-c")
+    template: str = simple_parsing.field(alias="-T")
+    json_mode: bool = simple_parsing.field(default=False, alias="-j")
```

Comment:
A `column_name` wasn't really enough. Now these two variables are shared between the two types and can potentially be moved to the parent class. I didn't do it yet, since future tasks might not follow the same structure.

Reply:
I tend to think we'll want to hoist these, but we can certainly do it later.